test/Generators/Microsoft.Gen.MetadataExtractor/GoldenReports/MeterDimensionsAttributedWithXmlDescriptions_LogMethod.json

```
@@ -1,71 +1,76 @@
                                 {
-                                    "Name": "C2",
-                                    "Notes": "Note 3"
+                                    "Name": "user",
+                                    "Type": "string",
+                                    "File": "src-0.cs",
+                                    "Line": "14",
+                                    "Classifications": [
+                                        {
+                                            "Name": "C2",
+                                            "Notes": "Note 3"
+                                        }
+                                    ]
+                                },
+                                {
+                                    "Name": "port",
+                                    "Type": "int",
+                                    "File": "src-0.cs",
+                                    "Line": "14"
                                 }
                             ]
-                        },
-                        {
-                            "Name": "port",
-                            "Type": "int",
-                            "File": "src-0.cs",
-                            "Line": "14"
                         }
                     ]
                 }
             ]
-        }
-    ]
-} , "MetricReport": [
- {
-  "TestClasses":
-  [
-    {
-     "MetricName": "DescribedDimensionCounter",
-     "MetricDescription": "(Missing Summary)",
-     "InstrumentName": "Counter",
-     "Dimensions": {
-      "Dimension1": "Dimension1 description.",
-      "Dim1": ""
-      }
-    },
-    {
-     "MetricName": "DescribedDimensionHistogram",
-     "MetricDescription": "(Missing Summary)",
-     "InstrumentName": "Histogram",
-     "Dimensions": {
-      "Dimension2": "Dimension2 description.",
-      "DimensionDefinedInMetricClass": "DimensionDefinedInMetricClass description."
-      }
-    },
-    {
-     "MetricName": "MyStrongTypeMetricWithDescription",
-     "MetricDescription": "(Missing Summary)",
-     "InstrumentName": "Counter",
-     "Dimensions": {
-      "AnotherDimension": "Gets or sets anotherDimension.",
-      "MetricEnum": "Gets or sets MetricEnum.",
-      "MetricEnum2": "Gets or sets MetricEnum2.",
-      "Dim2": "Gets or sets Dim2.",
-      "SomeDim": "Gets or sets SomeDim.",
-      "Dim4Struct": "Gets or sets Dim4Struct.",
-      "Dim5Struct": "Gets or sets Dim5Struct."
-      }
-    }
-  ]
- }
-] }
\ No newline at end of file
+        },
+    "MetricReport": 
+        [
+            {
+                "TestClasses":
+                [
+                    {
+                        "MetricName": "DescribedDimensionCounter",
+                        "MetricDescription": "(Missing Summary)",
+                        "Dimensions":
+                            {
+                                "Dimension1": "Dimension1 description."
+                            }
```

And here the object has an extra indentation 

```suggestion
                        "Dimensions":
                        {
                            "Dimension1": "Dimension1 description."
                        }
```

---

i have removed the indentations. However putting the open brace up above will require me to change also on Compliance because every json object is opened on a separated line.

---

do you mind changing the compliance ? it remained untouched